RecoLocalMuon/RPCRecHit/src/DTObjectMap.cc

```diff
@@ -10,14 +10,8 @@
 #include "RecoLocalMuon/RPCRecHit/src/DTObjectMap.h"
 #include "RecoLocalMuon/RPCRecHit/src/DTStationIndex.h"
 
-DTObjectMap::DTObjectMap(MuonGeometryRecord const& record) {
-  edm::ESHandle<RPCGeometry> rpcGeo;
-  record.get(rpcGeo);
-
-  edm::ESHandle<DTGeometry> dtGeo;
```

Comment:
If you remove this (useless, indeed) you can also remove the include of "Geometry/DTGeometry/interface/DTGeometry.h"

Reply:
done